packages/modules/src/modules/banners/usEoyMoment/components/UsEoyMomentBannerTicker.tsx

```diff
@@ -177,25 +167,23 @@ const UsEoyMomentBannerTicker: React.FC<UsEoyMomentBannerTickerProps> = ({
             <div>
                 <div css={soFarContainerStyles}>
                     <div css={soFarCountStyles}>
-                        {!isGoalReached && currencySymbol}
-                        {isGoalReached
-                            ? tickerSettings.copy.goalReachedPrimary
-                            : runningTotal.toLocaleString()}
+                        {currencySymbol}
+                        {runningTotal.toLocaleString()}
                     </div>
-                    <div css={countLabelStyles}>
-                        {isGoalReached
-                            ? tickerSettings.copy.goalReachedSecondary
-                            : tickerSettings.copy.countLabel}
+                    <div css={countLabelStyles(isGoalReached, false)}>
+                        {tickerSettings.copy.countLabel}
                     </div>
                 </div>
 
-                <div css={goalContainerStyles}>
+                <div css={goalContainerStyles(isGoalReached)}>
                     <div css={totalCountStyles}>
                         {currencySymbol}
-                        {isGoalReached ? runningTotal.toLocaleString() : goal.toLocaleString()}
+                        {isGoalReached
+                            ? `${runningTotal.toLocaleString()} goal reached`
+                            : goal.toLocaleString()}
                     </div>
-                    <div css={countLabelStyles}>
-                        {isGoalReached ? tickerSettings.copy.countLabel : 'our goal'}
+                    <div css={countLabelStyles(isGoalReached, true)}>
+                        {isGoalReached ? "But it's not too late to give" : 'our goal'}
```

Comment:
do we want to display `<RUNNING_TOTAL> goal reached` or `<GOAL> goal reached`? If we reach the 1.25 mil goal the copy makes it sound like it should say `1.25 mil goal reached` even if we're way past it

Reply:
Yeah here they're flipped around so we want `<GOAL> goal reached`